qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AdministrationPage.java

```diff
@@ -16,50 +16,51 @@
 public class AdministrationPage extends Page {
 
 	private static final String URL = "/admin/index.htm";
-	private static final By MANAGE_USERS = By.cssSelector("#content a[href='/openmrs/admin/users/users.list']");
-	private static final By MANAGE_MODULES = By.cssSelector("#legacyui-manageModules a");
-	private final static By MANAGE_VISIT_TYPES = By.cssSelector("#legacyui-manageVisitTypes a");
-	private static final By MANAGE_PROVIDERS = By.cssSelector("#content a[href='/openmrs/admin/provider/index.htm']");
-	private static final By MANAGE_PERSONS = By.cssSelector("#content a[href='/openmrs/admin/person/index.htm']");
-	private static final By MANAGE_HTMLFORMS_PAGE = By.cssSelector("#content a[href*='/module/htmlformentry/htmlForms.list']");
+	private static final By MANAGE_USERS_LINK = By.cssSelector("#content a[href='/openmrs/admin/users/users.list']");
+	private static final By MANAGE_ROLES_LINK = By.cssSelector("#legacyui-manageRoles > a");
```

Comment:
is the element retrievable via `By.cssSelector("#legacyui-manageRoles > a");`?

Reply:
yes it is but lemi adjust that a lil bit